css/index.css

```diff
@@ -12,9 +12,83 @@ header{
 .navlinks{
     width:500px;
     display:flex;
+    gap: 50px;
+    text-decoration: none;
     justify-content: space-evenly;
     list-style-type: none;
 }
+/*Smaller Screens mobiles
+change the font size (1st change) and color (2nd change) of your headers (1st element), 
+change the padding or margin (1st change) and background color (2nd change) of your unordered lists (2nd element),
+ and change the font size (1st change) 
+and font style (2nd change) of your name at the top of your page (3rd element)*/
+
+@media (min-width:600px){
+body{
+    background-color:burlywood;
+}
+.navigation{
+    background-color: yellowgreen;
+}
+a{
+    font-size: 1em;
+    text-decoration: none;
+}
+h1,h2,h3{
+    color:blue;
+    margin-bottom:auto;
+}
+h1{
+    font-size: x-large;
+    font-style: italic;
+}
+}
+
+/*Larger Screens tablets */
+@media (min-width:992px){
+    body{
+        background-color:beige;
+    }
+    .navigation{
+        background-color:darkgrey;
+    }
+    a{
+        font-size: 1.5em;
+        text-decoration: none;
+    }
+    h1,h2,h3{
+        color:blueviolet;
+        margin-bottom:auto;
+    }
+    h1{
+        font-size: large;
+        font-style:normal;
+    }
+    }
+
+    /*laptops,desktops*/
+    @media (min-width:1200px){
```

Comment:
I noticed that the About link in the section header disappears when the page is open to this size. I'd check how you're setting the width of the nav bar.

Here''s a tip from the assignment reqs: 

> When working with sizing, it's best to use percentages, REMs and EMs.. you can read more about those [here](https://www.linkedin.com/pulse/pixels-vs-em-rem-percent-when-why-use-each-milan-savov).